andela/technites-bn-frontend
# #168049956  Profile Information from Travel Request Page(UI)

#### What does this PR do?
Autofill some of the User's Profile Information
#### Description of Task to be completed?
Remember all the personal details entered and  present it as non-editable form on the next travel request initiation screen
#### How should this be manually tested?
1) clone repository
```shell
$ git clone https://github.com/andela/technites-bn-frontend.git
```
2) checkout in a feature branch
```shell
$ git checkout ft-autofill-profile-info-168049956
```
3) run tests
```shell
$  npm test
```
4) run the application
```shell
$  npm start
```
5) Open HTTP client (web browser) and navigate to `/request`  then make a request and remember to check `remember personal details checkbox` then on next requests those personal details are pre-filled automatically
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168049956](https://www.pivotaltracker.com/story/show/168049956)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
